react/global.css

```diff
@@ -1,6 +1,7 @@
 .vtex-product-kit__separator {
   width: 40px;
```

Comment:
tachyons? ❤️

Reply:
Tachyons has no definition for very specific sizes like this one.